backend/test/controllers/client.test.js

```diff
@@ -0,0 +1,71 @@
+var server = require('../../server.js')
+var request = require('supertest')
+var should = require('should')
+var client = require('../../models/client.js')
+var user = require('../../models/user.js')
+var crypto = require('crypto')
+
+describe('Client controller test', function () {
+  it('should add a client to the db', function (done) {
+    var password = 'testpassword'
+    var shasum = crypto.createHash('sha1')
+    shasum.update(password)
+    password = shasum.digest('hex')
+
+    user.build({username: 'testname', password: password, email: 'testemail'}).save().then(function (user) {
```

Comment:
put the instantiation of user as before? Repetitive creation of user in the tests below